src/main/resources/changesets/changelog-consent-94.0.xml

```diff
@@ -0,0 +1,34 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="94.0" author="connorlbark">
+        <!--Add mapping and dataset property type columns -->
+        <addColumn tableName="dataset">
+            <column name="dac_id" type="bigint">
+                <constraints foreignKeyName="fkDacId" nullable="true"/>
```

Comment:
Use standard camel case for constraint names: https://www.postgresql.org/docs/11/ddl-constraints.html
```suggestion
                <constraints foreignKeyName="fk_dac_id" nullable="true"/>
```